projects/plugins/jetpack/extensions/blocks/ai-paragraph/index.js

```diff
@@ -73,7 +73,11 @@ export const settings = {
 	edit,
 	save: attrs => {
 		const blockProps = useBlockProps.save();
-		return <div { ...blockProps }>{ attrs.attributes.content }</div>;
+		return (
+			<RawHTML { ...blockProps }>
+				{ attrs.attributes.content.trim().replaceAll( '\n', '<br/>' ) }
+			</RawHTML>
```

Comment:
What do you think about this?

```suggestion
			<RawHTML { ...blockProps }>
				<p>{ attrs.attributes.content.trim().replaceAll( '\n', '</p></p>' ) }
			</RawHTML>
```

Reply:
I have no special preferences. @artpi ?

Reply:
He said no :)

Reply:
Yeah I think that would mess up when transformed
It would create a paragraph blocks with nested <p> inside the paragraph which is not valid HTML syntax